### PR TITLE
CRUD application for patient record

### PR DESCRIPTION
**Pull request by Panas18**

### CODE REVIEW

File: src/controller/patientController.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+ * @param {Response} res
+ */
+export const getAllPatients = (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  patientService
+    .getAllPatients()
+    .then((data: any) => res.json(data))
```
Comment (aman007shrestha) [inline src/controller/patientController.ts:16]: Avoid using any type as much as possible. The essence of typescript lies in limiting types.

**Review by aman007shrestha — APPROVED** [review]
LGTM

File: src/controller/patientController.ts (hunk 1 above)
Comment (Panas18) [reply src/controller/patientController.ts:16]: Sure, I will correct it.

File: src/db/migrations/20220806104658_create_patient_account_table.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    table.string("dob").notNullable();
+    table.string("gender").notNullable();
+    table.string("email").unique().notNullable();
+    table.string("ethinicity").notNullable();
+    table.string("street").notNullable();
+    table.string("state").notNullable();
+    table.string("city").notNullable();
+    table.string("insurance_id").notNullable().unique();
+    table.string("member_id").notNullable();
+    table.string("insurance_provider").notNullable();
```
Comment (aman007shrestha) [inline src/db/migrations/20220806104658_create_patient_account_table.ts:17]: You can do table.date for dates table.integers for numbers.

**Pull request merged**